### PR TITLE
feat: github workflow for dependabot pr auto merge

### PR DESCRIPTION
**Pull request by rutujasurve94**
GitHub script as per:
https://github.com/marketplace/actions/dependabot-auto-merge
for auto-merging dependabot PRs labeled with `dependencies`, `github.actor` is dependabot and PR has `bump jinaai/jina` in the name.

### CODE REVIEW

**Comment by rutujasurve94** [comment]
confirming if `GITHUB_TOKEN` should be a part of the env like:
```
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
or a part of `with` as in the PR.

File: .github/workflows/merge-dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    types: [ labeled ]
+jobs:
+  auto-merge:
+    if: ${{ github.event.label.name == 'dependencies' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ahmadnassri/action-dependabot-auto-merge@v2
+        with:
+          target: minor
```
Comment (nan-wang) [inline .github/workflows/merge-dependabot.yml:14]: why `minor` here? I am expecting `patch`

File: .github/workflows/merge-dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+
+name: auto-merge
+on:
+  pull_request:
+    types: [ labeled ]
```
Comment (nan-wang) [inline .github/workflows/merge-dependabot.yml:5]: What is the motivation for this `types`?

**Review by nan-wang — CHANGES_REQUESTED** [review]

File: .github/workflows/merge-dependabot.yml (hunk 1 above)
Comment (rutujasurve94) [reply .github/workflows/merge-dependabot.yml]: Updated to `patch`.

File: .github/workflows/merge-dependabot.yml (hunk 2 above)
Comment (rutujasurve94) [reply .github/workflows/merge-dependabot.yml:4]: PRs from dependabot have the `labels` like `dependencies`, etc. This is to filter on only labeled PRs

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Comment by deepankarm** [comment]
@rutujasurve94 I don't think, this was ready to be merged. As discussed, please test it on your fork. The current merged action would skip all PRs.

**Pull request merged**